src/modules/train_controller/src/train_controller.cc

```diff
@@ -1,12 +1,15 @@
-#include "train_controller.h"
+#include <cstdint>
+#include <chrono>
 
-#include "convert.h"
+#include "train_controller.h"
```

Comment:
This is include should go first.  See [https://bte12.atlassian.net/wiki/spaces/NN/pages/1081369/C+Coding+Standard#Names-and-Order-of-Includes](https://bte12.atlassian.net/wiki/spaces/NN/pages/1081369/C+Coding+Standard#Names-and-Order-of-Includes)